caravel/assets/package.json

```diff
@@ -38,7 +38,6 @@
     "autobind-decorator": "^1.3.3",
     "bootstrap": "^3.3.6",
     "bootstrap-datepicker": "^1.6.0",
-    "bootstrap-toggle": "^2.2.1",
```

Comment:
Was this removed intentionally? It is still required in [explore.jsx](https://github.com/airbnb/caravel/blob/master/caravel/assets/javascripts/explore/explore.jsx#L21)